### ISSUE TITLE
`cryptocompare` test is unstable

### ISSUE THREAD

**Issue by gakonst** [interledger-rs/interledger-rs#501]
this just happened in https://circleci.com/gh/interledger-rs/interledger-rs/3163, unclear why

```
test cryptocompare ... FAILED

failures:

---- cryptocompare stdout ----
thread 'cryptocompare' panicked at 'called `Result::unwrap()` on an `Err` value: OperationError(())', src/libcore/result.rs:1084:5
stack backtrace:
   0:     0x55f7e975786b - backtrace::backtrace::libunwind::trace::h89fcc71e59e3bc5b
                               at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.34/src/backtrace/libunwind.rs:88
   1:     0x55f7e975786b - backtrace::backtrace::trace_unsynchronized::h0bad9be1379e729a
                               at /cargo/registry/src/github.com-1ecc6299db9ec823/backtrace-0.3.34/src/backtrace/mod.rs:66
   2:     0x55f7e975786b - std::sys_common::backtrace::_print::hd3382a1f33c473da
                               at src/libstd/sys_common/backtrace.rs:47
   3:     0x55f7e975786b - std::sys_common::backtrace::print::h0ec6f03cfb8e76a6
                               at src/libstd/sys_common/backtrace.rs:36
   4:     0x55f7e975786b - std::panicking::default_hook::{{closure}}::h96cbf7b454e3f557
                               at src/libstd/panicking.rs:200
   5:     0x55f7e97574dc - std::panicking::default_hook::h95a8f00337383d83
                               at src/libstd/panicking.rs:211
   6:     0x55f7e9757fdd - std::panicking::rust_panic_with_hook::h92f98b46e22f14ed
                               at src/libstd/panicking.rs:477
   7:     0x55f7e9757b62 - std::panicking::continue_panic_fmt::h25abfbb4e5b7043a
                               at src/libstd/panicking.rs:384
   8:     0x55f7e9757a46 - rust_begin_unwind
                               at src/libstd/panicking.rs:311
   9:     0x55f7e977932d - core::panicking::panic_fmt::h7e9f94035af782b3
                               at src/libcore/panicking.rs:85
  10:     0x55f7e9779427 - core::result::unwrap_failed::hf7591c1dd9412006
                               at src/libcore/result.rs:1084
  11:     0x55f7e8119b3e - core::result::Result<T,E>::unwrap::hd6960321faba4cd2
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libcore/result.rs:852
  12:     0x55f7e810213d - exchange_rates::cryptocompare::h3fbc2ac72cf37583
                               at crates/ilp-node/tests/exchange_rates.rs:149
  13:     0x55f7e81315da - exchange_rates::cryptocompare::{{closure}}::h3ff15b5232175e82
                               at crates/ilp-node/tests/exchange_rates.rs:86
  14:     0x55f7e810862e - core::ops::function::FnOnce::call_once::hbcf2edbecab8df84
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libcore/ops/function.rs:235
  15:     0x55f7e815f28f - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::hac0fbe2754f8bd24
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/liballoc/boxed.rs:787
  16:     0x55f7e976096a - __rust_maybe_catch_panic
                               at src/libpanic_unwind/lib.rs:80
  17:     0x55f7e817987e - std::panicking::try::h9d6938917cc05e3f
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panicking.rs:275
  18:     0x55f7e817987e - std::panic::catch_unwind::h5be70fd1d69515b0
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panic.rs:394
  19:     0x55f7e817987e - test::run_test::run_test_inner::{{closure}}::h056f206eafd8ff6e
                               at src/libtest/lib.rs:1408
  20:     0x55f7e8154cb5 - std::sys_common::backtrace::__rust_begin_short_backtrace::h859abbc742a34b19
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/sys_common/backtrace.rs:77
  21:     0x55f7e8158df5 - std::thread::Builder::spawn_unchecked::{{closure}}::{{closure}}::hf8192ec938c43694
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/thread/mod.rs:470
  22:     0x55f7e8158df5 - <std::panic::AssertUnwindSafe<F> as core::ops::function::FnOnce<()>>::call_once::hae7b2431555cb4db
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panic.rs:315
  23:     0x55f7e8158df5 - std::panicking::try::do_call::h6da7ecd0783259d7
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panicking.rs:296
  24:     0x55f7e976096a - __rust_maybe_catch_panic
                               at src/libpanic_unwind/lib.rs:80
  25:     0x55f7e81593d2 - std::panicking::try::h3c1c35414c2c78f1
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panicking.rs:275
  26:     0x55f7e81593d2 - std::panic::catch_unwind::ha5cfb2018fda983e
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/panic.rs:394
  27:     0x55f7e81593d2 - std::thread::Builder::spawn_unchecked::{{closure}}::hc7620e41f0938c09
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libstd/thread/mod.rs:469
  28:     0x55f7e81593d2 - core::ops::function::FnOnce::call_once{{vtable.shim}}::h1b3b20eeb6ea6ddd
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/libcore/ops/function.rs:235
  29:     0x55f7e97492af - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::h2877b2aefb0fd627
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/liballoc/boxed.rs:787
  30:     0x55f7e975fcb0 - <alloc::boxed::Box<F> as core::ops::function::FnOnce<A>>::call_once::h13f473dbcdaa0fef
                               at /rustc/625451e376bb2e5283fc4741caa0a3e8a2ca4d54/src/liballoc/boxed.rs:787
  31:     0x55f7e975fcb0 - std::sys_common::thread::start_thread::h562a7c4953cb1d31
                               at src/libstd/sys_common/thread.rs:13
  32:     0x55f7e975fcb0 - std::sys::unix::thread::Thread::new::thread_start::h382c6ae14181d163
                               at src/libstd/sys/unix/thread.rs:79
  33:     0x7f44fc224fa3 - start_thread
  34:     0x7f44fc1394cf - clone
  35:                0x0 - <unknown>


failures:
Nov 05 16:45:55.021  WARN interledger_store_redis::store: Could not issue psubscribe to Redis: Could not read enough bytes    
    cryptocompare

test result: FAILED. 1 passed; 1 failed; 0 ignored; 0 measured; 0 filtered out
```

**Comment by dora-gt** [comment]
This should be fixed by #503

**Comment by dora-gt** [comment]
I think this can be closed now because #503 is merged.
